server/modules/tasks.ts

```diff
@@ -0,0 +1,125 @@
+import { BaseServer } from '@logux/server/base-server'
+import {
+  defineChangedSyncMap,
+  defineChangeSyncMap,
+  defineCreatedSyncMap,
+  defineCreateSyncMap,
+  defineDeletedSyncMap,
+  defineDeleteSyncMap
+} from '@logux/actions'
+
+import { Task } from '../../api'
+import {
+  changeTask,
+  createTask,
+  deleteTask,
+  findTask,
+  getUserTasks
+} from '../db'
+
+const channel = 'tasks'
+
+export default (server: BaseServer): void => {
+  const createAction = defineCreateSyncMap<Task>(channel)
+  const changeAction = defineChangeSyncMap<Task>(channel)
+  const deleteAction = defineDeleteSyncMap(channel)
+
+  const createdAction = defineCreatedSyncMap<Task>(channel)
+  const changedAction = defineChangedSyncMap<Task>(channel)
+  const deletedAction = defineDeletedSyncMap(channel)
+
+  server.channel(channel, {
```

Comment:
You need to use `addSyncMap()` and `addSyncMapFilter()`

They are available Logux Server from GitHub (use `pnpm add @logux/server@logux/server`).